docs/source/tutorial.rst

```diff
@@ -724,3 +724,36 @@ sample points for both functions:
 
     print(f"Quadrature results: {integral1}, {integral2}")
 
+Multidimensional/Vectorized Integrands
+--------------------------------------
+
+If you wish to evaluate many different integrands over the same domain, it may be faster to pass in a vectorized formulation if possible.
```

Comment:
Maybe we could link here to the scipy thing you used? So people that are familiar with it realize that is the same way.

Reply:
Good call.  I'll add the link.